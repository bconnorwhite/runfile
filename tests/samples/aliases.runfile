# ----------
# Command Aliases
# ----------

# Simple aliases
b, build:
  cargo build

t, test, check:
  cargo test

# Aliases with arguments
d, deploy target:
  echo "Deploying to $target"

# Aliases with flags
r, run --debug:
  cargo run $debug
