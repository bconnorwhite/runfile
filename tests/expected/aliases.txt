Command Aliases
  b, build        # Simple aliases
  t, test, check
  d, deploy       # Aliases with arguments
    target
  r, run          # Aliases with flags
    --debug
